package.json

```diff
@@ -22,7 +22,7 @@
     "aframe-react": "^4.4.0",
     "bootstrap": "^4.0.0",
     "cannon": "^0.6.2",
-    "create-react-app": "^1.5.2",
+    "create-react-app": "^3.0.1",
```

Comment:
Was there a particular reason for updating this?

Reply:
I was having problems with create react app after installing eslint, so I had no remove it then install create-react-app again. I can try and go to an older version if needed